frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -4045,16 +4160,27 @@ gatherAndFilterCandidates(Context* context,
 
   // don't worry about last resort for compiler generated candidates
 
-  // next, look for candidates without using POI.
+  // look for candidates without using POI.
```

Comment:
TODO: These blocks are identical and repeated twice, I should make a helper function.